Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/ContentStreamForking/ForkContentStreamWithDisabledNodesWithoutDimensions.feature

```diff
@@ -56,15 +56,17 @@ Feature: On forking a content stream, hidden nodes should be correctly copied as
       | nodeVariantSelectionStrategy | "allVariants"      |
 
   Scenario: on ForkContentStream, the disabled nodes in the target content stream should still be invisible.
-    When the command ForkContentStream is executed with payload:
-      | Key                   | Value                |
-      | sourceContentStreamId | "cs-identifier"      |
-      | contentStreamId       | "user-cs-identifier" |
-
+    # Uses ForkContentStream implicitly
+    When the command CreateWorkspace is executed with payload:
+      | Key                | Value                |
+      | baseWorkspaceName  | "live"               |
+      | workspaceName      | "user-test"          |
+      | newContentStreamId | "user-cs-identifier" |
 
     # node aggregate occupation and coverage is not relevant without dimensions and thus not tested
 
-    When I am in content stream "user-cs-identifier" and dimension space point {}
+    When I am in workspace "user-test" and dimension space point {}
+    Then I expect the workspace to point to content stream "user-cs-identifier"
     And VisibilityConstraints are set to "withoutRestrictions"
     Then I expect node aggregate identifier "lady-eleonode-rootford" to lead to node user-cs-identifier;lady-eleonode-rootford;{}
```

Comment:
The `Then I expect the workspace to point to content stream` might be kinda obsolete as the `to lead to` step asserts the content stream id